src/window/apple.mm

```diff
@@ -732,6 +732,10 @@ - (NSString*) filePromiseProvider: (NSFilePromiseProvider*)filePromiseProvider f
     }
   }
 
+  SSC::String Window::getTitle () {
+    return SSC::String([this->window.title UTF8String]);
+  }
+
   void Window::setTitle (const SSC::String& seq, const SSC::String& value) {
```

Comment:
We could also store the title on the `Window` or `ManagedWindow` class instance too

Reply:
is there an opportunity for it to become out of sync though if we do that? this isn't in the performance path so I'd say its fine.